Bottom: fd438ca9665550ccb5bbf074f7d01ba3179bc91d
Top:    a79148579f602412211a86e896f1539dfe0f4d5d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:46:17 +0000

Refresh of add-virtual-trackball-code

---

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 01627a2..b0e6f31 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -30,6 +30,7 @@ src/hid/gtk/gui-log-window.c
 src/hid/gtk/gui-misc.c
 src/hid/gtk/gui-netlist-window.c
 src/hid/gtk/gui-top-window.c
+src/hid/gtk/gui-trackball.c
 src/main.c
 src/move.c
 src/netlist.c
